### ISSUE TITLE
[BUG] IconButton on denied consent screen is missing a label/tooltip

### ISSUE THREAD

**Issue by clocksmith** [joshua-s/coronavirus-diary#114]
**Describe the bug**
IconButton on denied consent screen is missing a label. It should add a tooltip string.

**To Reproduce**
0. Enabled VoiceOver/Talkback
1. Navigate to the denied consent screen by clicking the No button on the consent screen.
2. Traverse to the back icon

**Expected behavior**
The screen reader announces that the back arrow goes back to the consent screen.
